app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+import os
+from typing import Any, Callable
+from functools import wraps
+
+
+class CommandError(Exception):
+    """Checks command input for right format."""
+
+
+def check_command(func: Callable) -> Callable:
+    @wraps(func)
+    def wrapper(command: str) -> Any:
+        if command.split()[0] != "mv" or len(command.split()) != 3:
```

Comment:
You could perform the `split` once and do not use `split` twice in one if condition.

Reply:
should i use additional variable for this case?
e.g.:
 words = command.split()
        if words[0] != "mv" or len(words) != 3: